tests/ui_tests/app/page/cluster_list/page.py

```diff
@@ -34,6 +37,9 @@
     ActionDialog,
     DeleteDialog,
 )
+from tests.ui_tests.app.page.common.popups.locator import (
+    HostCreationLocators,
+)
```

Comment:
Is it some instrument's work of doing this? :D

Reply:
no, it's just pycharm) in case i need to import smthg else

Reply:
:D